### DISCUSSION TITLE
Dsiabling or altering the F7 insteract key

### DISCUSSION THREAD

**Discussion by AstralHeretic** [baffler/Transparent-Twitch-Chat-Overlay#233] [Q&A] [answered]
Is there any way to change or disable the F7 interaction key. I regularly use the function keys in games and never ran into problems with the programme using only F9 but now with the programme using multiple F keys its messed with a lot of setups I use.

**Comment by baffler** [comment, marked as answer]
This is no longer an issue with v1.0.1. I added Ctrl+Alt to all the default keys, but you can also change them now in the settings. It's under the General section there.

**Reply by AstralHeretic** [reply]
perfect. thanks for working on this <3
